templates/zerver/api/changelog.md

```diff
@@ -11,6 +11,11 @@ below features are supported.
 
 ## Changes in Zulip 5.0
 
+**Feature level 73**
+
+* [`POST /register`](/api/register-queue): Replaced `allow_community_topic_editing`
+  field with an integer field `edit_topic_policy`.
```

Comment:
We also need to document here the changes to `POST /realm`, even though it doesn't have `zulip.yaml` documentation yet; that'll help us ensure that when we do document that endpoint, we mention this change appropriately.

Reply:
Added this.